sample/lit-todo/package.json

```diff
@@ -5,10 +5,10 @@
     "@material/mwc-fab": "^0.18.0",
     "@material/mwc-icon": "^0.18.0",
     "@material/mwc-icon-button": "^0.18.0",
-    "@material/mwc-list": "^0.18.0",
+    "@material/mwc-checkbox": "^0.18.0",
     "@material/mwc-textfield": "^0.18.0",
     "@material/mwc-top-app-bar": "^0.18.0",
     "lit-html": "^1.3.0",
-    "replicache": "0.0.7"
+    "replicache": "../../"
```

Comment:
This didn't work for me but I was using `@web/dev-server`. Let me try with `es-dev-server`